src/core/utils/string.utils.ts

```diff
@@ -54,19 +54,26 @@ export const getWalletWidthForWallets = (wallets: BudgetStatementWalletDto[]) =>
   return '180px';
 };
 
-export const getCorrectRoleApi = (role: RoleUserDTO) => {
-  switch (role) {
-    case 'CoreUnitAdmin':
-      return 'Core Unit Admin';
-    case 'CoreUnitFacilitator':
-      return 'Core Unit Facilitator';
-    case 'SuperAdmin':
-      return 'Site Admin';
-    case 'User':
-      return 'User';
-    default:
-      return '';
+export const getCorrectRoleApi = (user: UserDTO) => {
+  const allPermission: string[] = [];
+
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  (user as any).roles.forEach((role: UserRole) => {
```

Comment:
if the `UserDTO` can have a `roles` property, is not better to add that to the dto instead of type cast the object so we can utilize the typescript checks here?

Reply:
You are right, lets move the roles to DTO